README.md

```diff
@@ -47,20 +46,20 @@ sequenceDiagram
 
 ## Features
 
-Currently, DAL supports CRUD operations on persisted objects, where data is
-persisted in Postgres database or in-memory cache, depending on library user's
-choice.
+Currently following features are suppotrd,
+persisted in Postgres database.
 
 - **CRUD operations** on persisted objects, where data is persisted in Postgres
-  database or in-memory cache, depending on library user's choice
+  database
 - **Role-based access control (RBAC)**.
+- Multi-tenancy based scoping for TENANT_READER and TENANT_WRITER roles.
```

Comment:
I think that **Multi-tenancy based scoping** would suffice, without mentioning the DB role names.
In my opinion, there is no need to go into detail describing which DB roles are tenant-specific in a section that just lists overall features supported by the library

Reply:
"Multi-tenancy based scoping" would need to be bold too, since the other 2 bullet points are already bold

Reply:
done